### PR TITLE
Dropped Node 12.

### PR DESCRIPTION
**Pull request by mrickard**
Signed-off-by: mrickard <maurice@mauricerickard.com>

## Proposed Release Notes
Updated the minimum version of the newrelic agent peer dependency to be `>=8.7.0`.

## Links
https://issues.newrelic.com/browse/NR-35638

## Details
Dropped Node 12 support.
Added JetBrains files to .gitignore.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,10 +34,10 @@
     "tap": "^16.0.1"
   },
   "engines": {
-    "node": ">=12.0"
+    "node": ">=14.0"
   },
   "peerDependencies": {
-    "newrelic": ">=6.11.0"
+    "newrelic": ">=8.7.0"
```
Comment (bizob2828) [inline package.json:40]: FYI: this is fine, but we will have to do another PR after we ship 9.x of the agent to update these

**Review by bizob2828 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mrickard) [reply package.json:40]: When do we have 9.x scheduled?

Comment (bizob2828) [reply package.json:40]: we typically don't schedule it but it's the Node 18 MMF. maybe later this week, or early next?

Comment (bizob2828) [reply package.json:40]: I'm going to revert this in the release prep
